packages/bddeb

```diff
@@ -177,8 +177,27 @@ def write_debian_folder_from_branch(root, templ_data, branch):
     )
 
 
-def read_version():
-    return json.loads(run_helper("read-version", ["--json"]))
+def read_version_from_meson():
```

Comment:
Note @ogayot that packages/bddeb  is called directly by [this branch PR CI Build package step](https://github.com/canonical/cloud-init/actions/runs/18810737941/job/53671504968?pr=6539) beyond just the manual test procedures I mentioned in the pull request description. 

bddeb and brpm are considered developer-only tooling to assist building local debs or rpms for testing. They are 'examples' which official downstreams can peek at, but they are definitely not production code. They are however used in CI and our Jenkins instance also calls into bddeb in [server-jenkins-jobs](https://github.com/canonical/server-jenkins-jobs/blob/main/jenkins-jobs/cloud-init/integration.yaml#L178)